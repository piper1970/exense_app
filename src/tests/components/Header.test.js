import {shallow} from 'enzyme';
import React from 'react';
import Header from '../../components/Header';

describe('Header', () => {
    test('should render header correctly', () => {
        const wrapper = shallow(<Header />);
        expect(wrapper).toMatchSnapshot();
    });
});