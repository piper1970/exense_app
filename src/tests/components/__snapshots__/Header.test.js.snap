// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header should render header correctly 1`] = `
<header>
  <h1>
    Expensify
  </h1>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/"
  >
    Dashboard
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/add"
  >
    Add Expense Page
  </NavLink>
  <NavLink
    activeClassName="is-active"
    ariaCurrent="true"
    exact={true}
    to="/help"
  >
    Help Page
  </NavLink>
</header>
`;
